### PR TITLE
Updated Octopus Get Usage script

### PR DESCRIPTION
**Pull request by ryanrousseau**
---

# Pre-requisites

- [x] `Id` should be a **GUID** that is not `00000000-0000-0000-0000-000000000000`
  - **NOTE** If you are modifying an existing step template, please make sure that you **do not** modify the `Id` property *(updating the `Id` will break the Library sync functionality in Octopus)*. 
- [x] `Version` should be incremented, otherwise the integration with Octopus won't update the step template correctly
- [x] Parameter names should not start with `$`
- [x] **Step template parameter names (the ones declared in the JSON, not the script body) should be prefixed with a namespace so that they are less likely to clash with other user-defined variables in Octopus** (see [this issue](https://github.com/OctopusDeploy/Issues/issues/2126)). For example, use an abbreviated name of the step template or the category of the step template).
- [x] `LastModifiedBy` field must be present, and (_optionally_) updated with the correct author
- [x] The best practices documented [here](https://github.com/OctopusDeploy/Library/wiki/Best-Practices) have been applied
- [ ] If a new `Category` has been created:
   - [ ] An image with the name `{categoryname}.png` must be present under the `step-templates/logos` folder
   - [ ] The `switch` in the `humanize` function in [`gulpfile.babel.js`](https://github.com/OctopusDeploy/Library/blob/master/gulpfile.babel.js#L92) must have a `case` statement corresponding to it

### CODE REVIEW

File: step-templates/octopus-get-octopus-usage.json [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
...ts)\n`t`t`tSSH Target Count: $($ObjectCounts.DisabledSshTargets)\n`t`t`tKubernetes Target Count: $($ObjectCounts.DisabledKubernetesCount)\n`t`t`tAzure Web App Target Count: $($ObjectCounts.DisabledAzureWebAppCount)\n`t`t`tAzure Service Fabric Cluster Target Count: $($ObjectCounts.DisabledAzureServiceFabricCount)\n`t`t`tAzure (Legacy) Cloud Service Target Count: $($ObjectCounts.DisabledAzureCloudServiceCount)\n`t`t`tAWS ECS Cluster Target Count: $($ObjectCounts.DisabledECSClusterCount)\n`t`t`tOffline Target Count: $($ObjectCounts.DisabledOfflineDropCount)\n`t`t`tCloud Region Target Count: $($ObjectCounts.DisabledCloudRegions)\n`t`t`tFtp Target Count: $($ObjectCounts.DisabledFtpTargets)\n`tWorker Count: $($ObjectCounts.WorkerCount)\n`t`tActive Workers: $($ObjectCounts.ActiveWorkerCount)\n`t`tUnavailable Workers: $($ObjectCounts.UnavailableWorkerCount)\n`t`tWorker Breakdown\n`t`t`tListening Tentacle Target Count: $($ObjectCounts.ListeningTentacleWorkers)\n`t`t`tPolling Tentacle Target Count: $($ObjectCounts.PollingTentacleWorkers)\n`t`t`tSSH Target Count: $($ObjectCounts.SshWorkers)\n\"@\n\nWrite-Host $text\n$text > octopus-usage.txt\nNew-OctopusArtifact \"$($PWD)/octopus-usage.txt\""
```
Comment (mcasperson) [inline step-templates/octopus-get-octopus-usage.json:13]: The variable name `ActiveListeningTentacleTargets` seems like a contradiction. It's not part of this PR, but looking at the code I think the name of this variable should have been `PassiveListeningTentacleTargets`. Or maybe even just `ListeningTentacleTargets`, as `Active` and `Passive` here are synonyms of `Polling` and `Listening`.

**Review by mcasperson — APPROVED** [review]
This change looks simple enough, although some of the variable names confused me when I tried to read the logic.

**Review by twerthi — APPROVED** [review]
LGTM
